packages/outputarea/src/widget.ts

```diff
@@ -1124,9 +1124,13 @@ export class Stdin extends Widget implements IStdin {
 
     this._input = this.node.getElementsByTagName('input')[0];
     // make users aware of the line history feature
-    this._input.placeholder = this._trans.__(
-      '↑↓ for history. Search history with c-↑/c-↓'
-    );
+    if (!this._password) {
+      this._input.placeholder = this._trans.__(
+        '↑↓ for history. Search history with c-↑/c-↓'
+      );
+    } else {
+      this._input.placeholder = this._trans.__('');
```

Comment:
Thank you! I think an empty string here does not need a translation:

```suggestion
      this._input.placeholder = '';
```